README.md

```diff
@@ -354,6 +354,51 @@ pip install jina[hub] && jina hub new --type app
 
 This creates a Python entrypoint, YAML configs and a Dockerfile. You can start from there.
 
+## Multi & Cross Modality
+
+Jina is a data type agnostic framework. It basically can work with any data type. 
```

Comment:
```suggestion
Jina is a data type-agnostic framework. It basically can work with any data type. 
```

Reply:
are we sure? data type go together, it seems weird. @alexcg1 what would be correct English?

Reply:
![image](https://user-images.githubusercontent.com/8330330/101018987-92c42400-356c-11eb-908f-4c361ea30efd.png)
There are certain words, that, when used as suffixes, require a dash (`-`)

Reply:
![image](https://user-images.githubusercontent.com/8330330/101019161-d61e9280-356c-11eb-9b4f-091d77cd771c.png)

It's a v minor thing :P

Reply:
Yes, my point is then datatype belongs together?